recipes/cosapp_lab/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "cosapp_lab" %}
+{% set version = "0.15.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f7c1b9a9466e2d64792b7e1b5933af1b6b3af5d402b9c83b5e2be00da9658680
+  
+build:
+  noarch: python
+  number: 0
+  skip_compile_pyc:
+    - "*/utils/templates/*.py" 
+  entry_points:
+    - cosapp = cosapp_lab.main:cosapp
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

These options are the default when using conda-build
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv"
```

---

updated